### ISSUE TITLE
Parameters for reproducing Ablation

### ISSUE THREAD

**Issue by zshn25** [xjixzz/vadepth-net#3]
Dear authors,

I am trying to reproduce your ablation table 4, with height loss on monodepth2 network. I ran with your default parameters, and `height_loss_weight 0.001` but did not manage to get a scale of 1. 

Could you please share your parameters for this experiment?

Thank you

**Comment by xjixzz** [comment]
Thanks for your interest in our paper. I have uploaded the [weights](https://drive.google.com/file/d/1q0AGeyl9vs9gN6BAs--jzMwradsalnwc/view?usp=sharing) of  monodepth2 trained with the absolute depth loss.   I think the reason for your training failure might be the training strategy. To train monodepth2 with the absolute depth loss, we just follow the original training strategy of monodepth2 (lr is set as 1e^-4 for the first 15 epochs and then dropped to 1e^-5 for the last 5 epochs) instead of the default training strategy of vadepth-net. Maybe you can try to change the schedule for learning rate.

**Comment by xjixzz** [comment]
By the way, the height_loss_weight should be set as 0.01 instead of 0.001.

**Comment by zshn25** [comment]
Thank you for the weights. I tried again with your suggestions. The best I could get was abs_rel of 0.114 with scale 1.24.

Maybe the library versions of Pytorch and so on matter. Did you use monodepth2‘s suggested versions?

**Comment by xjixzz** [comment]
The abs_rel of your results is close to ours but the scale is not. The mean of median scales of ours is 1.032, which is much closer to unity than yours.  I retrained monodepth2 with the absolute depth loss and still got the scale of 1.034.

I am not sure whether the environment setting leads to the differences. I did not use monodepth2's env settings. I have provided our python envs in the README.md. Maybe you can try the newer version of pytorch following our instructions.
